enterprise/dev/ci/ci/pipeline-steps.go

```diff
@@ -192,7 +192,7 @@ func addBackendIntegrationTests(c Config) func(*bk.Pipeline) {
 func addBrowserExtensionE2ESteps(pipeline *bk.Pipeline) {
 	for _, browser := range []string{"chrome", "firefox"} {
 		// Run e2e tests
-		pipeline.AddStep(fmt.Sprintf(":%s:", browser),
+		pipeline.AddStep(fmt.Sprintf(":%s: Extension E2E", browser),
```

Comment:
"Extension" can be ambiguous (Sourcegraph extension or browser extension?).
```suggestion
		pipeline.AddStep(fmt.Sprintf(":%s: Browser extension e2e tests", browser),
```

Reply:
I can add "Browser extension" to make it more clear, I left it to just "extension" as the icon sort of shows the browser, like in `:go: Test`.

Reply:
```suggestion
		pipeline.AddStep(fmt.Sprintf(":%s: Browser extension E2E", browser),
```

Reply:
Can we use the `browser` variable in place of the "Browser" word?

Reply:
I can do `E2E for %s extension` as the `browser` variable is lowercase. I could also use a func to capitalize it, but it seems overkill for this.

Reply:
I was curious if Go had a stdlib function for this, looks like it does: `strings.Title()`. Anyway agree that it's not as important.

Reply:
Following this in https://github.com/sourcegraph/sourcegraph/pull/15354#discussion_r518182194